samples/Ralltiir/Ralltiir.Server/Startup.cs

```diff
@@ -0,0 +1,191 @@
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Antiforgery;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.SpaServices.ReactDevelopmentServer;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+using OpenIddict.Abstractions;
+using Ralltiir.Server.EF;
+using Ralltiir.Server.Models;
+
+namespace Ralltiir.Server
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddSwaggerGen(c =>
+            {
+                c.SwaggerDoc("v1", new OpenApiInfo {Title = "Ralltiir.Server", Version = "v1"});
+                c.ResolveConflictingActions(apiDescriptions => apiDescriptions.First());
+            });
+
+            services.AddMvc();
+
+            services.AddAntiforgery(options =>
```

Comment:
Please add a comment indicating why we need antiforgery and why we have to customize that.

Reply:
done ^_^